Sources/StytchUI/StytchB2BUIClient/Managers/AuthenticationOperations.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import StytchCore
 
-struct PasswordAuthenticateResetManager {
+struct AuthenticationOperations {
```

Comment:
The naming of those struct is something I am still struggling with. Its basically a place to put what would otherwise be called "hooks" but I like it all in one place so I am not sure how to structure this in code in a way that makes sense, open to suggestions here.